api-firewall/content.md

```diff
@@ -0,0 +1,80 @@
+%%LOGO%%
+
+# What is API Firewall?
+
+Light-weighted Wallarm API Firewall protects your API endpoints in cloud-native environments with API schema validation. Wallarm API Firewall relies on a positive security model allowing calls that match a predefined API specification, while rejecting everything else.
+
+## API schema validation and positive security model
+
+When starting API Firewall, you should provide the [OpenAPI 3.0 specification](https://swagger.io/specification/) of the application to be protected with API Firewall. The started API Firewall will operate as a reverse proxy and validate whether requests and responses match the schema defined in the specification.
+
+The traffic that does not match the schema will be logged using the [`STDOUT` and `STDERR` Docker services](https://docs.docker.com/config/containers/logging/) or blocked (depending on the configured API Firewall operation mode). If operating in the logging mode and detecting the traffic on endpoints that are not included in the specification, API Firewall also logs these endpoints as the shadow ones (except for endpoints returning the code `404`).
+
+Provided API schema should be described using the [OpenAPI 3.0 specification](https://swagger.io/specification/) in the YAML or JSON file (`.yaml`, `.yml`, `.json` file extensions).
+
+By allowing you to set the traffic requirements with the OpenAPI 3.0 specification, Wallarm API Firewall relies on a positive security model.
+
+## Technical characteristics
+
+API Firewall works as a reverse proxy with a built-in OpenAPI 3.0 request and response validator. The validator is written in Go and optimized for extreme performance and near-zero added latency.
+
+
+# How to use this image
+
+## Quick start with Docker
+
+This quick start guide walks through downloading and starting Wallarm API Firewall with minimal configuration on Docker.
+
+### Requirements
+
+* [Installed and configured Docker](https://docs.docker.com/get-docker/)
+* [OpenAPI 3.0 specification](https://swagger.io/specification/) developed for the REST API of the application that should be protected with Wallarm API Firewall
+
+### Up and running
+
+To run Wallarm API Firewall with minimal configuration on Docker:
+
+1. Create the Docker network to connect your application and Wallarm API Firewall (e.g. `api-firewall-network`):
+
+    ```bash
+    docker network create api-firewall-network
+    ```
+2. Start the containerized application to be protected with API Firewall (e.g. [kennethreitz/httpbin](https://hub.docker.com/r/kennethreitz/httpbin/)):
+
+    ```bash
+    docker run --rm -it --network api-firewall-network \
+        --network-alias backend -p 8090:8090 kennethreitz/httpbin
```

Comment:
Usually we'd suggest this should be another official image, but given how popular this example is (and how it's just a stand-in for anything else), I'm not going to press it. :+1: